voila/execute.py

```diff
@@ -129,5 +129,7 @@ def executenb(nb, cwd=None, km=None, **kwargs):
     resources = {}
     if cwd is not None:
         resources['metadata'] = {'path': cwd}  # pragma: no cover
+    if "config" not in kwargs and km is not None:
+        kwargs["config"] = km.config
```

Comment:
Great idea, however, I'd rather see https://github.com/QuantStack/voila/blob/6a1d9793b769ff3aad929b0497b9dcd4b6f41569/voila/handler.py#L58 modified to:
```
result = executenb(notebook, km=km, cwd=cwd, config=self.exporter_config)
```
And rename `export_config` to `config`.

Reply:
Moved to #240. To check, you were suggesting `s/self.exporter_config/self.config/`?